mitodl/mit-open
# Shanbady/ingest sloan events

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4744

### Description (What does it do?)
Adds etl pipeline to Ingest upcoming sloan webinars

### How can this be tested?
1. checkout this branch
2. open a django shell and run the pipeline 
```
from news_events.etl.pipelines import sloan_webinars_etl
sloan_webinars_etl()
```
4. go to the homepage and verify that the sloan webinars show up in "upcoming events section" and that the links work.

### Additional Context
The format for dates in the in upcoming webinars api is not in some standardized format. To get around this we try parsing the "Image_Text" attribute using a natural language date parser and then fallback to the "published date" if we cant parse it.
